src/Assembly_Binario/Binario.java

```diff
@@ -6,135 +6,315 @@
 	public Binario(){
 		
 	}
+
+	private char tipoInstrucao;
 	
 	public String Assembly_Para_Binario(String opcao) {		
 		switch(opcao) {
 			case "sll" :
-				return "000 0000";	
+				setTipoInstrucao('R');
+				return "000000";
 			case "srl" :
+				setTipoInstrucao('R');
+				return "000010";
 			case "j" :
-				return "000 0010";
+				setTipoInstrucao('J');
+				return "000010";
 			case "jal" :
+				setTipoInstrucao('J');
+				return "000011";
 			case "sra" :
-				return "000 0011";
+				setTipoInstrucao('R');
+				return "000011";
 			case "beq" :
+				setTipoInstrucao('I');
+				return "000100";
 			case "sllv" :
-				return "000 0100";
+				setTipoInstrucao('R');
+				return "000100";
 			case "bne" :
-				return "000 0101";
+				setTipoInstrucao('I');
+				return "000101";
 			case "blez" :
+				setTipoInstrucao('');
```

Comment:
Em todos os campos que for retornar vazio da um espaço para ele interpretar que é um carácter char preenchido.